### ISSUE TITLE
[A11y] narrator does not announce "Required" on some inputs when IsRequired is true.

### ISSUE THREAD

**Issue by dhoehna** [microsoft/AdaptiveCards#8915]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

DevHome

### Problem Description

Narrator does not read say an item is required when setting "IsRequired" to true on a Input.ChoiceSet.

### Screenshots

![image](https://github.com/microsoft/AdaptiveCards/assets/2517139/c9744e6f-c9d3-4d54-b796-f7ced7a4e103)

Narrator does not mention if any combo box is required.  To validate, I removed "Value" from the adaptive card JSON to not have a pre-selected value.  Narrator still did not say "Required"

### Card JSON

```JSON
{
  "type": "AdaptiveCard",
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.5",
  "body": [
    {
      "type": "ColumnSet",
      "spacing": "large",
      "columns": [
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Input.Text",
              "id": "NewEnvironmentName",
              "label": "${EnterNewEnvironmentTextBoxLabel}",
              "maxLength": 100,
              "isRequired": true,
              "errorMessage": "${DevBoxTextBoxErrorMessage}",
              "regex": "${DevBoxNameRegex}",
              "spacing": "large",
              "value": "${TextBoxText}"
            }
          ]
        }
      ]
    },
    {
      "type": "ColumnSet",
      "spacing": "large",
      "columns": [
        {
          "type": "Column",
          "width": "stretch",
          "items": [
            {
              "type": "Input.ChoiceSet",
              "id": "ProjectsComboBox",
              "label": "${ProjectComboBoxLabel}",
              "devHomeChildChoiceSetId": "PoolsComboBox",
              "devHomeSelectionChangedDataForChildChoiceSet": "${SelectionChangedDataForChildChoiceSet}",
              "devHomeRefreshChildChoiceSetOnSelectionChanged": true,
              "isRequired": true,
              "errorMessage": "${DevBoxProjectComboBoxErrorMessage}",
              "style": "compact",
              "value": "${SelectedProjectIndex}",
              "choices": [
                {
                  "$data": "${ProjectList}",
                  "title": "${title}",
                  "value": "${value}"
                }
              ]
            }
          ]
        }
      ]
    },
    {
      "type": "ColumnSet",
      "spacing": "large",
      "columns": [
        {
          "type": "Column",
          "width": "stretch",
          "items": [

            {
              "type": "Input.ChoiceSet",
              "id": "PoolsComboBox",
              "devHomeParentChoiceSetId": "ProjectsComboBox",
              "label": "${DevBoxPoolComboBoxLabel}",
              "isRequired": true,
              "errorMessage": "${DevBoxPoolsComboBoxErrorMessage}",
              "placeholder": "${DevBoxPoolsPlaceHolder}",
              "style": "compact",
              "value": "${SelectedPoolIndex}",
              "devHomeChoicesData": [
                {
                  "$data": "${PoolsList}",
                  "title": "${title}",
                  "subtitle": "${subtitle}",
                  "value": "${value}"
                }
              ]
            }
          ]
        }
      ]
    },
    {
      "type": "ActionSet",
      "id": "DevHomeTopLevelActionSet",
      "actions": [
        {
          "id": "DevHomeMachineConfigurationNextButton",
          "type": "Action.Submit",
          "title": "${PrimaryButtonLabelForCreationFlow}"
        },
        {
          "id": "DevHomeMachineConfigurationPreviousButton",
          "type": "Action.Submit",
          "title": "${SecondaryButtonLabelForCreationFlow}"
        }
      ]
    }
  ]
}
```
